src/index.css

```diff
@@ -1,27 +1,135 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
 body {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  font-size: 14px;
+  font-weight: 100;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+button {
+  display: block;
+  background: none;
+  border: none;
+  padding: 0;
+  font-family: inherit;
+  user-select: none;
+  cursor: pointer;
+  outline: none;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+#wrapper {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#root {
+  width: 320px;
+  height: 520px;
+  position: relative;
+}
+
+.calc {
+  width: 100%;
+  height: 100%;
+  background: black;
+  display: flex;
+  flex-direction: column;
+}
+
+#wrapper .calc {
+  box-shadow: 0 0 20px 0 #aaa;
+}
+
+.calc-display {
+  color: white;
+  background: #1c191c;
+  line-height: 130px;
+  font-size: 6em;
+  flex: 1;
+}
```

Comment:
- [ ] Your design is more or less fine and it sticks with the original design in many aspects. Yet the display field looks like it needs a little more work to match it well with the original design. Particularly, the background color and the font size. :thinking: 
![image](https://user-images.githubusercontent.com/64894927/129636910-950ba1ab-7a5d-46eb-bfe7-bf1476057123.png)
![image](https://user-images.githubusercontent.com/64894927/129636940-7deaba1e-1eca-423e-b65a-738c42858f21.png)